.github/workflows/performance.yaml

```diff
@@ -0,0 +1,45 @@
+name: Performance Benchmark
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  hyperfine:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            npm-${{ hashFiles('package-lock.json') }}
+            npm-
+      - name: Environments
+        run: |
+          echo
+          echo Using Node $(node -v), NPM $(npm -v)
+          echo
+      - name: Install Dependencies
+        run: npm ci
+      - name: Install External
+        run: npm i -g hyperfine @redocly/cli@latest
```

Comment:
should it compare with the latest or with the version from main?

I think latest is fine for now but would be great to check-out main branch and compare it to it somehow in the future.

Reply:
Yes, comparing to main would be better, but I haven't found a simple and reliable solution to this yet. Will check how this approach works first, and then we can decide how to evolve it.